lib/wibox/layout/align.lua

```diff
@@ -266,6 +441,9 @@ end
 --   widgets are then given the remaining space on either side.
 --   If the center widget requires all available space, the outer widgets are
 --   not drawn at all.
+-- * `"justified"`: The widgets in the slot one and three are set to the same
```

Comment:
I don't know where to put this comment, so let's say this line is the place to open the discussion:

I don't like this much to manipulate literal strings, and I think an "enum like table" to maps the available modes would be more user-friendly. I did one for the mouse buttons, for example https://github.com/awesomeWM/awesome/blob/392dbc21ab6bae98c5bab8db17b7fa7495b1e6a5/lib/awful/button.lua#L42-L48.